src/components/Comment/Comment.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { CommentType } from '../types/CommentType';
+import './Comment.scss';
+
+export const Comment = ({ body, email, name }) => (
+  <>
+    <hr />
+    <p>{body}</p>
+    <div className="comment__user">
+      <span>{name}</span>
+      <span>{email}</span>
```

Comment:
Думаю лучше будет указать имеил ссылкой и задать href

```suggestion
      <a href={`mailto:${email}`} className="comment__user-email">
        {email}
      </a>
```